web-client/public/js/createNetwork.js

```diff
@@ -144,11 +144,14 @@ containing "-", "_", and alpha-numeric characters only`);
             grnState.customWorkbook.sources = response.sources;
             grnState.customWorkbook.source = Object.keys(response.sources).length === 1 ?
                 Object.keys(response.sources)[0] : null;
+            let i;
+            let dateTime;
+            let timestamp;
             for (let source in response.sources) {
-                let i = source.indexOf(":");
-                let date = new Date(source.substring(i + 1));
-                date = getFormattedDateTime(date);
-                grnState.customWorkbook.sources[source].timestamp = date;
+                i = source.indexOf(":");
+                dateTime = new Date(source.substring(i + 1));
+                timestamp = getFormattedDateTime(dateTime);
+                grnState.customWorkbook.sources[source].timestamp = timestamp;
```

Comment:
I see what you did here but…why keep the `let`? These variables are not used outside of the block scope of the `for` loop, so there is no need to define them outside. Also, in this new version, variables aren’t re-assigned, so overall this is the cleanest option, I’d say:
```suggestion
                const i = source.indexOf(":");
                const dateTime = new Date(source.substring(i + 1));
                const timestamp = getFormattedDateTime(dateTime);
                grnState.customWorkbook.sources[source].timestamp = timestamp;
```

(removing all the preceding `let`s of course)